tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -1,10 +1,23 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#ifdef OE_USE_OPENSSL
```

Comment:
It looks like this test is written to require two separate build lines.  It would be more efficient in my opinion if the same test binary could test both openssl and mbedtls, thus not requiring CI/CD to build everything twice.

Reply:
It's required to build twice as the OE library can link against one of the crypto wrapper implementations at one time.

Reply:
That's not very efficient.   Just like we have plugins/modules for other things, why can't we do the same for this?  At worst, I think it should be a link-time, not compile-time, choice, right?

Reply:
Refer to the [design](https://github.com/openenclave/openenclave/blob/master/docs/DesignDocs/enclave_openssl_support.md#user-experience). Yes, it's an link-time option, but per design, we cannot link against both implementation at the same time---both mbed- and openssl-based implementation are used for the same set of crypto APIs.

Reply:
In the same CI/CD build, you could link it twice (with different resulting binary names) and not have to compile everything twice.  However, it would be better in the future for it to be a runtime choice not a link-time choice.
This is not a blocker, but currently this complicates CI/CD and slows down every bors run because it has to compile it both ways and test it both ways, so doubles the cost.

Reply:
Done.